packages/strapi-generate-api/templates/bookshelf/service.template

```diff
@@ -121,7 +121,7 @@ module.exports = {
     const data = _.omit(values, <%= globalID %>.associations.map(ast => ast.alias));
 
     // Create entry with no-relational data.
-    const entry = <%= globalID %>.forge(params).save(data);
+    const entry = await <%= globalID %>.forge(params).save(data);
```

Comment:
I mean there is no need to await here because the entry itself is not used afterward.
By applying this change you're for the save to be finished before updating the relations (Which might be good actually)

Reply:
No, it's absolutely should be here. 
I had issue because of that. I wasn't able to update record more than once. U can check it. So it's a small fix, for a huge problem.
Reasons:
- if error will occur here, request will be successful anyway.
- order of execution interrupted.
- someone may try to use this instance, and he will be surprised, that there just promise, instead of entry.

BTW, u can check other templates, there await keyword exists.